src/MigrationTools/Configuration/EngineConfigurationBuilder.cs

```diff
@@ -22,13 +22,26 @@ public EngineConfigurationBuilder(ILogger<EngineConfigurationBuilder> logger)
 
         public EngineConfiguration BuildFromFile(string configFile = "configuration.json")
         {
-            string configurationjson;
-            using (var sr = new StreamReader(configFile))
-                configurationjson = sr.ReadToEnd();
-            var ec = JsonConvert.DeserializeObject<EngineConfiguration>(configurationjson,
-                    new FieldMapConfigJsonConverter(),
-                    new ProcessorConfigJsonConverter(),
-                    new MigrationClientConfigJsonConverter());
+            EngineConfiguration ec = null;
+            try
+            {
+                string configurationjson;
+                using (var sr = new StreamReader(configFile))
```

Comment:
Perhaps change to just File.ReadToEnd?
https://docs.microsoft.com/en-us/dotnet/api/system.io.file.readalltext?view=netcore-3.1